core/src/main/java/com/novoda/noplayer/SubtitleView.java

```diff
@@ -0,0 +1,82 @@
+package com.novoda.noplayer;
+
+import android.content.Context;
+import android.graphics.Canvas;
+import android.util.AttributeSet;
+import android.view.View;
+
+import com.novoda.noplayer.model.NoPlayerCue;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public final class SubtitleView extends View {
```

Comment:
Still now that you have implemented your own view you can create a contract with
```
void setCues(List<NoPlayerCue> cues);
void setVisibility(int visible);
```

So that we can interchange the view with a different implementation or let the client to use its own view